src/components/task-step/ends.cjsx

```diff
@@ -49,14 +50,14 @@ PracticeEnd = React.createClass
 
     # custom footer for practices
     footer =
-      <div className='-practice-end'>
+      <div className="-#{type}-end">
```

Comment:
Could/should these be `data-type={type}`? That way we're not dynamically creating class names? (greppability). Maybe I'm the only one though ; (